rnd/autogpt_builder/src/app/layout.tsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import type { Metadata } from "next";
-import { ThemeProvider as NextThemeProvider } from "next-themes";
-import { type ThemeProviderProps } from "next-themes/dist/types";
 import { Inter } from "next/font/google";
```

Comment:
bad indentation in this file

Reply:
Oh thank you for noticing that. I've fixed the tab error on line 52. 

In a future PR would we like to standardize this with a prettier implementation?

Reply:
Yeah that makes sense. Can you open an issue for it?